integration/spark/src/main/scala/org/apache/spark/sql/execution/strategy/CarbonDataSourceScan.scala

```diff
@@ -49,9 +50,19 @@ case class CarbonDataSourceScan(
     directScanSupport: Boolean,
     @transient extraRDD: Option[(RDD[InternalRow], Boolean)] = None,
     tableIdentifier: Option[TableIdentifier] = None,
+    @transient selectedCatalogPartitions : Seq[CatalogTablePartition] = Seq.empty,
+    @transient partitions: Seq[SparkExpression],
```

Comment:
what are the different points between "partitionFilters" and "partitions"?

Reply:
partitionFilters - does not have Dynamic subquery expression. Renamed the variables